packages/fleather/lib/src/widgets/editor.dart

```diff
@@ -910,8 +912,28 @@ class RawEditorState extends EditorState
       ),
       cause,
     );
+
     if (cause == SelectionChangedCause.toolbar) {
-      bringIntoView(textEditingValue.selection.extent);
+      switch (defaultTargetPlatform) {
```

Comment:
Isn't this weird? On master we have this change but it compares it with the old version. @amantoux

Reply:
I fail to understand to what you mean by 'it compares it with the old version'?